File: performance/go/00/main.go
package main

import (
    "fmt"
    "time"
)

func main() {
    start := time.Now()
    var sum int64 = 0
    for i := int64(1); i <= 1_000_000_000; i++ {
        sum += i
    }
    duration := time.Since(start)
    fmt.Printf("Sum: %d, Time: %v\n", sum, duration)
}

File: performance/rust/00/main.rs
use std::time::Instant;

fn main() {
    let start = Instant::now();
    let mut sum: i64 = 0;
    for i in 1..=1_000_000_000 {
        sum = sum.wrapping_add(i);
    }
    let duration = start.elapsed();
    println!("Sum: {}, Time: {:?}", sum, duration);
}
